include/Dialect/LWE/IR/LWEAttributes.td

```diff
@@ -19,7 +19,19 @@ class LWE_EncodingAttr<string attrName, string attrMnemonic, list<Trait> traits
   let assemblyFormat = "`<` struct(params) `>`";
 }
 
-def LWE_BitFieldEncoding: LWE_EncodingAttr<"BitFieldEncoding", "bit_field_encoding"> {
+class LWE_EncodingAttrWithScalingFactor<string attrName, string attrMnemonic, list<Trait> traits = []>
+    : LWE_EncodingAttr<attrName, attrMnemonic, traits> {
+  let parameters = (ins
+    // These two parameters identify a scaling factor to scale cleartexts in
+    // preparation for noise growth of FHE schemes. In this format they are
+    // restricted to being powers of two.
```

Comment:
This makes it seem as if `cleartext_start` and `cleartext_bitwidth` are powers of two (but instead they represent the log of the scaling factor or the bit index of the cleartext) - maybe a slight reword like "The parameters are the log of the scaling factor, which restricts the scaling factor to powers of two"

Reply:
Done